test/parallel/test-fs-stat-abort-test.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+require('../common');
+const test = require('node:test');
+const assert = require('node:assert');
+const fs = require('node:fs');
+
+const filePath = './temp.txt';
+fs.writeFileSync(filePath, 'Test');
+
+test('fs.stat should throw AbortError when abort signal is triggered', async () => {
+  const controller = new AbortController();
+  controller.abort();
+  const { signal } = controller;
```

Comment:
```suggestion
  const signal = AbortSignal.abort();
```

That said, this should fail immediately when `fs.stat` is called rather than interrupting it since the signal is already triggered. To test properly this should try to abort immediately after calling `fs.stat`... unfortunately it's likely that the operation completes so quickly that it might be flaky. Might need to carefully think through how to more effectively test.

Reply:
thank you so much for sharing this great information and support 🙏